Dockerfile

```diff
@@ -8,4 +8,6 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 EXPOSE 8080
 
+ENV TEQUILA_API_KEY=1QTVqV0VjqZPZbyiiXl3WenpNSCCmnnE
```

Comment:
Use dotenv instead

Reply:
To address the comment, we need to use the `python-dotenv` package to load the API key from a `.env` file instead of hardcoding it in the Dockerfile. 

Here are the steps to make the changes:

1. Add `python-dotenv` to the `requirements.txt` file.
2. Create a `.env` file and add the API key to it.
3. Modify the `api.py` and `main.py` files to load the API key from the `.env` file using `python-dotenv`.
4. Update the Dockerfile to copy the `.env` file to the Docker container.
5. Update the unit tests in the `tests/test_api.py` and `tests/test_main.py` files to load the API key from the `.env` file.

Here is a summary of the changes:

1. Add `python-dotenv` to the `requirements.txt` file.
2. Create a `.env` file and add the API key to it.
3. Modify the `api.py` and `main.py` files to load the API key from the `.env` file using `python-dotenv`.
4. Update the Dockerfile to copy the `.env` file to the Docker container.
5. Update the unit tests in the `tests/test_api.py` and `tests/test_main.py` files to load the API key from the `.env` file.